src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs

```diff
@@ -27,15 +28,16 @@ public class TrainingCoursesController : ControllerBase
         }
 
         [HttpGet]
-        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null)
+        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null, [FromQuery]int orderBy = 0)
         {
             try
             {
                 var queryResult = await _mediator.Send(new GetTrainingCoursesListQuery
                 {
                     Keyword = keyword, 
                     RouteIds = routeIds,
-                    Levels = levels
+                    Levels = levels,
+                    OrderBy = (OrderBy)orderBy
```

Comment:
having a string from front end can be mapped to the web's enum, and then map to inner api enum. don't assume that they will map 0==0, 1==1. be explicit. else there'll be mysterious bugs one day when a new order by is added or removed.

Reply:
This is also fixed